### PR TITLE
DDCE-5033 Play 3 Migration

### PR DESCRIPTION
**Pull request by robpetterson**

### CODE REVIEW

**Comment by robpetterson** [comment]
Have kept sbt-accessibility-linter on version  0.38 as there are some issues updating CSRF in the template which is necessary and would require an additional ticket in the future.

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,24 +2,25 @@ import sbt.*
...
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "commons-validator"            % "commons-validator"           % "1.8.0",
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-28" % "8.5.0",
-    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapPlayVersion,
-    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % hmrcMongoPlayVersion,
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30" % "8.5.0",
+    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-30"         % hmrcMongoPlayVersion,
     "commons-codec"                % "commons-codec"               % "1.16.0",
```
Comment (gastonracui) [inline project/AppDependencies.scala:13]: upgrade to 1.16.1

File: project/CodeCoverageSettings.scala [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import sbt.Setting
+import scoverage.ScoverageKeys.{coverageHighlighting, _}
+
+object CodeCoverageSettings {
+  val settings: Seq[Setting[?]] = Seq(
+    coverageExcludedFiles := "<empty>;Reverse.*;.*filters.*;.*components.*;.*repositories.*;" +
```
Comment (gastonracui) [inline project/CodeCoverageSettings.scala:6]: you just need 

coverageExcludedFiles := "<empty>;.*components.*;.*Routes.*;",

here

**Review by gastonracui — CHANGES_REQUESTED** [review]

**Review by gastonracui — CHANGES_REQUESTED** [review]

File: project/CodeCoverageSettings.scala (hunk 2 above)
Comment (gastonracui) [reply project/CodeCoverageSettings.scala]: ".*components.*;.*repositories.*;.*Routes.*",

**Review by gastonracui — APPROVED** [review]

**Pull request merged**